index.html

```diff
@@ -51,33 +58,185 @@ <h1 class="about-title padding">I'm Rose Glad to see you!</h1>
+        <ul class="project-links">
+          <li class="canopy">CANOPY</li>
+          <li class="small-circle">.</li>
+          <li class="career-role">Back End Dev</li>
+          <li class="small-circle">.</li>
+          <li class="year">2015</li>
+        </ul>
+        <div class="project-description">
+          <p>A daily selection of privately personalized
+            reads; no accounts or sign-ups required.</p>
+        </div>
+        <ul class="my-skills">
+          <li><a href="#" class="skill-link html-btn">html</a></li>
+          <li><a href="#" class="skill-link css-btn">css</a></li>
+          <li><a href="#" class="skill-link js-btn">javascript</a></li>
+        </ul>
       
+        <a href="#" class="see-project-btn">See Project</a>
+      </div>
+
+      <section class="about-rose">
+        <div class="about-title-section">
+          <h1 class="about-title padding">About Myself</h1>
+        </div>
+        <div class="full-description">
+          <p class="about-me padding">Hello I’m a software developer! 
+            I can help you build a product , feature or website 
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don’t hestiate to contact me.
+          </p>
+        </div>
+        <div class="socials-section">
+          <button class="connect-link padding">LET'S CONNECT</button>
+          <ul class="icons">
+            <li><img src="./images/twitter.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/Linkedin.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/mail.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/github.svg" class="icon" alt="twitter icon"></li>
+            <li><img src="./images/playboy.svg" class="icon" alt="twitter icon"></li>
+          </ul>
+        </div>
+        <a href="#" class="see-project-btn">Get my resume</a>
+      </section>
+
+      <section class="languages">
+        <h4 class="languages-title">Languages</h4>
+        <ul class="card-language">
+          <li class="js-img">
+            <img src="./images/Frame9.svg" alt="Javascript">
+          </li>
+
+          <li class="html-img">
+            <img src="./images/Frame10.svg" alt="HTML">
+          </li>
+
+          <li class="css-img">
+            <img src="./images/Frame11.svg" alt="CSS">
+          </li>
+        </ul>
+      </section>
+
+      <section class="frameworks">
+        <div class="framework-btn">
+          <h4 class="languages-title">Frameworks</h4>
+          <a href="#"><img src="./images/Union.svg" alt="CSS"></a>
+        </div>
+        <hr>
+      </section>
+
+      <section class="skills">
+        <div class="skill-btn">
+          <h4 class="languages-title">Skills</h4>
+          <a href="#"><img src="./images/Union.svg" alt="CSS"></a>
+        </div>
+        <hr>
+      </section>
     </section>
```

Comment:
**A-** Please consider making your design in accordance with the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=37%3A2238&t=UPJfUKzuGg7shdH7-0). Currently, there are some problems with it as following:-

1- Margins and padding.
2- Fonts sizes etc.
2- Background image shape ( Needs to be curved from top-right corner)

**B-** Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`

Following is the code representation of the above if you are confused about what the instructions mean:-


`    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Python</li>
          <li>Java</li>
        </ul>
      </li>
      <li>Frameworks</li>
      <li>Skills</li>
    </ul>`


![image](https://user-images.githubusercontent.com/94223367/219422805-c6dd3bc7-bbc1-4dba-9c18-85bc5d823350.png)